.github/workflows/release_asset.yml

```diff
@@ -0,0 +1,51 @@
+name: Release asset bot
```

Comment:
Is this file an integral part of this PR and should really be committed (why?)

Reply:
This code is from another [PR](https://github.com/move-language/move/pull/208) and I'm waiting to be merged

Reply:
Understood, but including this code here make reviewing the current PR harder as it's not always clear which parts implement functionality this PR is aiming for and which are unrelated.

Reply:
Ok, I will try not to submit these two files